queries/liquid/highlights.scm

```diff
@@ -0,0 +1,111 @@
+([
+  "{{"
+  "}}"
+  "{{-"
+  "-}}"
+  "{%"
+  "%}"
+  "{%-"
+  "-%}"
+] @tag.delimiter
+  (#set! priority 101))
```

Comment:
Since it's a template language, there were instances where the highlight group of the embedded language would conflict with injected "parent" language -- IE. it would prioritize the JavaScript highlights over the Liquid highlights when some Liquid appeared in a script tag.

Maybe this isn't the right approach? It seemed sensible to me though.

Reply:
I see. We should have a consistent approach for such template languages then.

At the very least, the priority should be higher to indicate that this is cross-language (110, in analogy for the documented value of 90 for captures that injections should override).

The different priority 102 further down should not be needed then; instead use pattern order ("last wins").

Reply:
(Note also that this approach can have unintended side effects with languages that are injected into, e.g., css.)

Reply:
Right - I went ahead and changed the priorities to 110 from 101.

I tried going with 110 from 102 using the "last wins" approach, but for some reason I cannot not get the desired result.. Basically what I'm trying to achieve there is to have the tag delimiters for comments highlight as comments. The comments query is already below the delimiter query and I'm frankly quite confused as to why the the delimiters seem to still take priority.

For the time being I left the comments and raw statement 1 higher for consistency but I'll see what I can do to remove.

Reply:
I was able to fix this by making some small adjustments in the parser